companion-api/api/claims.go

```diff
@@ -0,0 +1,96 @@
+package api
+
+import (
+	"net/http"
+	"strings"
+
+	restful "github.com/emicklei/go-restful"
+	"github.com/mcluseau/autorizo/api"
+	"github.com/mcluseau/autorizo/auth"
+)
+
+// Register provide a restful.WebService from this API
+func (cApi *CompanionAPI) registerClaims(ws *restful.WebService) {
```

Comment:
what we want is a user API to allow administrators to change, not an API where users can change themselves.

Reply:
I see the tricks. No need to authenticate with jwt now. It's an admin API.